### PR TITLE
Possibility to choose custom runner image in integration tests

### PR DESCRIPTION
**Pull request by excitoon**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Possibility to choose custom runner image in integration tests. Can be useful for diagnostics.

### CODE REVIEW

File: tests/integration/runner [hunk 1]
```diff
@@ -28,8 +28,6 @@ CONFIG_DIR_IN_REPO = "programs/server"
 INTEGRATION_DIR_IN_REPO = "tests/integration"
 SRC_DIR_IN_REPO = "src"
 
-DIND_INTEGRATION_TESTS_IMAGE_NAME = "clickhouse/integration-tests-runner"
```
Comment (devcrafter) [inline tests/integration/runner:31]: Let's leave an ability to set default value via env variable.

File: tests/integration/runner [hunk 2]
```diff
@@ -226,6 +224,13 @@ if __name__ == "__main__":
         help="Set network driver for runnner container (defaults to `host`)",
     )
 
+    parser.add_argument(
+        "--docker-image",
+        action="store",
+        default="clickhouse/integration-tests-runner",
```
Comment (devcrafter) [inline tests/integration/runner:230]:
```suggestion
        default=DIND_INTEGRATION_TESTS_IMAGE_NAME,
```

**Review by devcrafter — CHANGES_REQUESTED** [review]
Thanks for PR. Please check the comments

**Comment by alexey-milovidov** [comment]
Abandoned.

**Pull request closed without merging**